actions/ci-publish-npm/action.yml

```diff
@@ -0,0 +1,50 @@
+name: ci-publish-npm
+description: Publishes to NPM
+
+inputs:
+  package-json-directory:
+    description: "The sub directory of the package.json if not at the root of the repository."
+    required: false
+    default: '.'
+  publish-command:
+    description: "The package.json defined script used to publish. Will run as `pnpm <publish-command>`"
+    required: true
+  npm-token:
+    description: The NPM token with publish access
+    required: true
+  github-token:
+    description: The GITHUB_TOKEN for the workflow
```

Comment:
should we say something like what level of perm it requires in order to run the `action-gh-release`?